Xamarin.Forms.Controls.Issues/Xamarin.Forms.Controls.Issues.Shared/Issue2448.xaml.cs

```diff
@@ -0,0 +1,75 @@
+﻿using System;
+using Xamarin.Forms.CustomAttributes;
+using Xamarin.Forms.Internals;
+
+namespace Xamarin.Forms.Controls.Issues
+{
+	[Preserve(AllMembers = true)]
+	[Issue(IssueTracker.Github, 2448, "Setting FlowDirection of Alerts and ActionSheets", PlatformAffected.iOS | PlatformAffected.Android)]
+
+	public partial class Issue2448 : ContentPage
+	{
+		public Issue2448()
+		{
+#if APP
+			InitializeComponent();
+#endif
+		}
+
+		void Button1_Clicked(object sender, EventArgs args)
```

Comment:
I think all of these `#if APP` can just be wrapped around this entire set of Clicked operations 

Also can you change the names of these to match what action they are causing?